src/engine/server/sv_init.cpp

```diff
@@ -82,11 +82,49 @@ void SV_SetConfigstring( int index, const char *val )
 	sv.configstringsmodified[ index ] = true;
 }
 
+static void SendConfigStringToClient( int cs, client_t *cl )
+{
+	char buf[ 1024 ]; // escaped characters, in a quoted context
+	// max command size for SV_SendServerCommand is 1022, leave a little overhead for the command
+	char *limit = buf + 990;
```

Comment:
FWIW, the cs chunking code uses 1024 - 64 as the size. Does it make sense to be consistent with that?

Reply:
No